src/libostree/ostree-sysroot.c

```diff
@@ -338,6 +340,19 @@ ensure_sysroot_fd (OstreeSysroot *self, GError **error)
   return TRUE;
 }
 
+static gboolean
+validate_boot_fd (OstreeSysroot *self, int fd, GError **error)
+{
+  g_assert_cmpint (fd, !=, -1);
+  struct statfs stbuf;
+  if (fstatfs (fd, &stbuf) < 0)
+    return glnx_throw_errno_prefix (error, "fstatfs");
```

Comment:
don't we want to know which fstatfs failed ? 
```suggestion
    return glnx_throw_errno_prefix (error, "fstatfs(boot_fd)");
```

Reply:
Sure, done